src/script/engines/lua.c

```diff
@@ -177,6 +266,11 @@ struct mScriptEngineContext* _luaCreate(struct mScriptEngine2* engine, struct mS
 	lua_getglobal(luaContext->lua, "require");
 	luaContext->require = luaL_ref(luaContext->lua, LUA_REGISTRYINDEX);
 
+	int status = luaL_dostring(luaContext->lua, _socketLuaSource);
+	if (status) {
+		printf("dostring: %s\n", lua_tostring(luaContext->lua, -1));
+	}
```

Comment:
Don't use printf. I should check if there's an mLOG category for scripting and add it if not.

Reply:
Hmm... You know, I was just intending to remove that. It was just there for debugging. But if you think mLOG is a better idea, then sure.

Reply:
_mLOG_CAT_SCRIPT btw